### PR TITLE
Project: Update notarization_tool.sh to take a key instead of username/password

### PR DESCRIPTION
**Pull request by russellhancox**

### CODE REVIEW

File: Conf/Package/package_and_sign.sh [hunk 1]
```diff
@@ -30,7 +30,7 @@ function die {
 [[ -n "${INSTALLER_SIGNING_KEYCHAIN}" ]] || die "INSTALLER_SIGNING_KEYCHAIN unset"
 
 # NOTARIZATION_TOOL is a required environment variable pointing to a wrapper
-# tool around the tool to use for notarization. The tool must take 2 flags:
+# tool around the tool to use for notarization. The tool must take 1 flags:
```
Comment (pmarkowsky) [inline Conf/Package/package_and_sign.sh:33]: Nit: flag vs flags

**Review by pmarkowsky — DISMISSED** [review]
LGTM

File: Conf/Package/package_and_sign.sh (hunk 1 above)
Comment (russellhancox) [reply Conf/Package/package_and_sign.sh]: Done

**Review by pmarkowsky — APPROVED** [review]
LGTM

**Pull request merged**